src/mri/exp_mult_mex.jl

```diff
@@ -0,0 +1,158 @@
+using Random
+
+include("max_percent_diff.jl")
```

Comment:
as part of MIRT now you don't need "include" here - just add the commented line:

# using MIRT: max_percent_diff